spec/models/story_spec.rb

```diff
@@ -115,4 +115,13 @@
       end
     end
   end
+
+  describe ".approved" do
+    it "returns only approved stories" do
+      approved_stories = FactoryBot.create_list(:story, 2, :approved)
+      FactoryBot.create_list(:story, 2, :rejected)
+
+      expect(Story.approved).to eq(approved_stories)
```

Comment:
Creating a scope wasn't needed because it is an `enum` attribute and Rails automagically generates those scopes for you

https://github.com/fastruby/points/blob/d0eb2eed881338a83a68a792a68c6c22106eea7c/app/models/story.rb#L11